test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperExecutionContextTest.cs

```diff
@@ -148,6 +148,7 @@ public void ParentItems_SetsItemsProperty()
             }
         }
 
+        [Theory]
```

Comment:
Ya hah, when I was fixing it up I was like why isn't the world exploding? :smile: